File: python/fizzbuzz.py
#!/usr/bin/python

for i in range(1,101):

  three = i % 3
  five  = i % 5
  
  if three == 0 and five == 0:
    print 'FizzBuzz'
  elif three == 0:
    print 'Fizz'
  elif five == 0:
    print 'Buzz'
  else:
    print i

File: go/fizzbuzz.go
package main

import "fmt"

func main() {

    for i := 1; i <= 100; i++ {

        var(
            fizz = i % 3
            buzz = i % 5
        )

        if fizz == 0 && buzz == 0 {
          fmt.Println("FizzBuzz")
          continue
        }

        if fizz == 0 {
          fmt.Println("Fizz")
          continue
        }

        if buzz == 0 {
          fmt.Println("Buzz")
          continue
        }

        fmt.Println(i)

    }

}
